### DISCUSSION TITLE
Why does the file I uploaded become a folder?

### DISCUSSION THREAD

**Discussion by moumou1234657890** [minio/minio#18306] [Q&A] [answered]
Hi.
Why does the file I uploaded become a folder? And if I create a file in minio, I can't view my file with the "mc ls". What can I do to see the file I created via the mc ls command?
version：RELEASE.2022-10-24T18-35-07Z

**Comment by klauspost** [comment, marked as answer]
`mc cat ALIAS/BUCKET/path...`

You should never use backend files directly.
